tools/generate-packed-assets

```diff
@@ -48,8 +48,7 @@ BUILD_CACHE="plugin AssetPack => { pipes => [qw(${ASSETPACK_PLUGINS})]} and app-
 BUILD_CACHE_OPTS='-Ilib/ -MMojolicious::Lite'
 
 generate() {
-    # shellcheck disable=SC2039
-    for _ in {1..3}; do
+    for _ in $(seq 1 3); do
```

Comment:
Why not just use Bash here by changing the shebang? Again, it makes sense to avoid spawning a sub process.